.github/workflows/testnet.yml

```diff
@@ -22,6 +24,26 @@ jobs:
           rustup default 1.81.0
           rustup update
       - run: cargo version
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: "temurin" # See 'Supported distributions' for available options
+          java-version: "19"
+
+      - name: Install apalache for e2e
+        run: |
+          gh release download --repo apalache-mc/apalache --pattern apalache.tgz
+          tar -xvzf apalache.tgz --directory . > /dev/null
+          echo "`pwd`/apalache/bin" >> $GITHUB_PATH
```

Comment:
Where is GITHUB_PATH used? Maybe add a comment?

Reply:
done